massa-node/base_config/gas_costs/abi_gas_costs.json

```diff
@@ -1,11 +1,11 @@
 {
-  "assembly_script_abort": 0,
+  "assembly_script_abort": 1,
```

Comment:
Does it make sence to have the abort be at 1? It's probably fine but I'm not sure how it all fits ;). E.g if abort is auto called when out of gas or something weird like that.